tests/e2e/compare/compare.ts

```diff
@@ -53,20 +58,16 @@ function buildCompareEntry(name, compare, baseline) {
 
 /**
  * Compare results between baseline and current entries and categorize.
- *
- * @param {Object} compareEntries
- * @param {Object} baselineEntries
- * @returns {Object}
  */
-function compareResults(compareEntries, baselineEntries) {
+function compareResults(compareEntries: Metric, baselineEntries: Metric) {
     // Unique test scenario names
-    const names = [...new Set([..._.keys(compareEntries), ..._.keys(baselineEntries || {})])];
+    const compareKeys = Object.keys(compareEntries);
+    const baselineKeys = baselineEntries ? Object.keys(baselineEntries) : [];
+    const names = Array.from(new Set([...compareKeys, ...baselineKeys]));
 
-    const compared = [];
-    const added = [];
-    const removed = [];
```

Comment:
Why were them removed?

Reply:
This was removed because this was not being used at all, these even were always empty.